Cubical/M-types/helper.agda

```diff
@@ -0,0 +1,317 @@
+{-# OPTIONS --cubical --guardedness #-} --safe
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Equiv using (_≃_)
+open import Cubical.Foundations.Function using (_∘_)
+
+open import Cubical.Data.Unit
+open import Cubical.Data.Prod
+open import Cubical.Data.Nat as ℕ using (ℕ ; suc ; _+_ )
+open import Cubical.Data.Sigma
+
+open import Cubical.Foundations.Transport
+open import Cubical.Foundations.Univalence
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.GroupoidLaws
+open import Cubical.Foundations.Path
+open import Cubical.Foundations.Embedding
+open import Cubical.Foundations.FunExtEquiv
+
+module Cubical.M-types.helper where
+
+module helper where
+
+refl-fun : ∀ {ℓ} {A : Set ℓ} (x : A) → x ≡ x
+refl-fun x = refl {x = x}
+
+----------------
+-- Iso helper --
```

Comment:
All but `funExt-iso`  in this section should definitely be in `Foundations.Isomorphism`, though possibly modulo renaming to be consistent with `Foundations.Equiv` (e.g. `sym-iso` -> `invIso`). `funExt-iso` should probably also be in `Functions.FunExtEquiv`?

Reply:
I will try and rename the theorem to match the naming in the files :) , and see where they will fit in. I will have a look at this tomorrow.